starter-code/apple-pie-starter-code/index.html

```diff
@@ -2,34 +2,52 @@
 <html lang="en">
 <head>
   <title>Apple Pie Recipe</title>
+  <link rel="stylesheet" type="text/css" href="style.css">
 </head>
 <body>
-Apple Pies
-This was my grandmother's apple pie recipe. I have never seen another one
+	<div class="apple">
+	<div>
+
+		<img src="apple-pie.jpg" class="apple-pie" >
+	</div>
+	<p class="apple-pie-text">Apple Pies</p>
+	<div class="grandmother">
+
+<div>This was my grandmother's apple pie recipe. I have never seen another one
 quite like it. It will always be my favorite and has won me several first place
-prizes in local competitions. I hope it becomes one of your favorites as well!
+prizes in local competitions. I hope it becomes one of your favorites as well!</div>
+<center><img src="recipe-info.png" class="recipe"></center>
```

Comment:
You can use a `div` instead `center` and set the `text-align: center` style to this `div` .